scrapy/exporters.py

```diff
@@ -96,9 +100,13 @@ class JsonItemExporter(BaseItemExporter):
     def __init__(self, file, **kwargs):
         self._configure(kwargs, dont_fail=True)
         self.file = file
-        self.encoder = ScrapyJSONEncoder(**kwargs)
+        self.encoder = ScrapyJSONEncoder(ensure_ascii=self.ensure_ascii, **kwargs)
         self.first_item = True
 
+    def _configure(self, options, dont_fail=False):
+        self.ensure_ascii = options.pop('ensure_ascii', True)
```

Comment:
Sorry, I meant that you'd need to pass the option [when the feedexporter initializes exporters](https://github.com/scrapy/scrapy/blob/bf7f67549378269c3976afc89abcf9c2190d242f/scrapy/extensions/feedexport.py#L188).
Or maybe I'm missing some bit?
I'm still new to exporters code